resources/nheko.appdata.xml.in

```diff
@@ -2,7 +2,7 @@
 <!-- Copyright 2020-2022 mujx, nheko reborn developers -->
 <component type="desktop">
   <id>@APPID@.desktop</id>
-  <launchable type="desktop-id">@APPID@.desktop</launchable>
+  <launchable type="desktop-id">im.nheko.Nheko.desktop</launchable>
```

Comment:
Doesn't this break, because the normal .desktop file is called nheko.desktop? Or does that only matter for the `<id>`?

Reply:
@deepbluev7 i'm so sorry, I meant to change the <id> field.